src/Testing/Concerns/MakesAssertions.php

```diff
@@ -139,15 +139,15 @@ protected function testEmitted($value, $params)
         $assertionSuffix = '.';
 
         if (empty($params)) {
-            $test = collect($this->payload['effects']['emits'])->contains('event', '=', $value);
+            $test = collect($this->payload['effects']['emits'] ?? [])->contains('event', '=', $value);
```

Comment:
`Collection` accepts also null, so i would pull stuff from a neasted-array like this `collect(data_get($this->payload, 'effects.emits'))`

applies for all appearances in this file

Reply:
Hot tip.. thanks